platform/broadcom/sonic-platform-modules-dell/s5248f/scripts/s5248f_platform.sh

```diff
@@ -86,6 +86,26 @@ switch_board_modsel() {
 	done
 }
 
+install_python_api_package() {
+    device="/usr/share/sonic/device"
+    platform=$(/usr/local/bin/sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)
+
+    rv=$(pip install $device/$platform/sonic_platform-1.0-py2-none-any.whl)
```

Comment:
Soon, the master branch will be moving to Debian Bullseye, and Python 2 support will be completely removed. The 202012 branch should have support for using the Python 3 version of the sonic-platform package, if it exists. Therefore, I suggest only installing the Python 3 version of the package. Otherwise, you will need to remove the installation of the Python 2 package in the near future on the master branch.

Reply:
Removed the Python2 support and tested with Python3 package.